MPF.md

```diff
@@ -326,7 +326,7 @@ This [augments file][Augments] will defines `trigger_upgrade` on hosts with IPv4
     "trigger_upgrade": [
       "ipv4_10_10_1",
       "ipv4_10_10_2",
-      "cfengine_3_10_(?!2$)\d+"
+      "cfengine_3_10_(?!2$)\\d+"
```

Comment:
Nice catch!

Reply:
actually, github pointed those out in red! :O